src/remediations/fifi.js

```diff
@@ -46,7 +46,28 @@ async function fetchRHCClientId (systems) {
     const systemIds = _.map(systems, system => { return system.id; });
     const systemProfileDetails = await inventory.getSystemProfileBatch(systemIds);
 
-    _.forEach(systems, system => system.rhc_client = systemProfileDetails[system.id].system_profile.rhc_client_id);
+    _.forEach(systems, system => {
+        system.rhc_client = systemProfileDetails[system.id].system_profile.rhc_client_id;
+        system.marketplace = systemProfileDetails[system.id].system_profile.is_marketplace;
+    });
+}
+
+function rhcAdditionalSorting (partitionedSystems) {
+    const noSmartManagement = _.filter(partitionedSystems[1], system => _.isUndefined(system.rhc_client) && !system.marketplace);
+    const rhcNotConfigured = _.filter(partitionedSystems[1], system => !_.isUndefined(system.rhc_client) && system.marketplace);
```

Comment:
Is this ever going to be non-empty given that `partitionedSystems` is partitioned by this same predicated on line 68?